modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -19,6 +19,20 @@ function acquia_cms_headless_install($is_syncing) {
 
     // Programmatically create a new "headless" user.
     $headless_user->createHeadlessUser();
+    // Add roles to headless user.
+    $roles = \Drupal::entityTypeManager()->getStorage('user_role')->loadMultiple([
+      'content_administrator',
+      'content_author',
+      'content_editor',
+    ]);
+    if ($roles) {
+      foreach ($roles as $role) {
```

Comment:
@rajeshreeputra This overall doesn't look correct to me. 

1. User should be loaded outside of foreach loop.
2. Why did we add `break` statement as that won't add other role to user.
3. User save should be outside of foreach loop.

Reply:
Updated.